app/models/fe/concerns/choice_field_concern.rb

```diff
@@ -105,7 +105,7 @@ def validation_class(answer_sheet)
       end
     end
 
-    def display_response(app=nil)
+    def display_response(app = nil, joiner = ', ')
```

Comment:
It's because display_response is used when reviewing.. Ie to pull and
format the answer when a reviewer views the app. So commas seemed more
natural there, even if they are used in the options themselves.
On 28 Oct 2015 10:04, "Josh Starcher" <notifications@github.com> wrote:

> In app/models/fe/concerns/choice_field_concern.rb
> <https://github.com/CruGlobal/fe/pull/57#discussion_r43258761>:
>
> > @@ -105,7 +105,7 @@ def validation_class(answer_sheet)
> >        end
> >      end
> >
> > -    def display_response(app=nil)
> > +    def display_response(app = nil, joiner = ', ')
>
> Why did you set the default to ',' as opposed to ';'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CruGlobal/fe/pull/57/files#r43258761>.
>